src/lib/components/Footer/FooterConfig.tsx

```diff
@@ -0,0 +1,235 @@
+import { PreLoginFooterLinksType, FooterLinksType, CompanyLinkType } from '@pagopa/mui-italia';
```

Comment:
Ask on Slack for all of these TODO in order to retrieve the links to use, asking if the links are global or product focused